### ISSUE TITLE
Import Lutris games

### ISSUE THREAD

**Issue by Owen-Benner** [tkashkin/GameHub#395]
Lutris has a large library of install scripts to automatically apply necessary tweaks to games. Currently, there's no quick way to take advantage of this in GameHub. If there were a way to install games in Lutris and then import them to GameHub, that would be very useful. Alternatively, if GameHub could install games using Lutris scripts, that would also be great (and probably even better in many cases).

**Comment by Bloodaxe95** [comment]
This would be great, since some of my games seem to run fine in Lutris but not in GameHub for some reason.

**Comment by Kellegram** [comment]
Yes x10000. GameHub is already fantastic, but so far I still have it installed alongside Lutris for that same reason. If GameHub could take that extra step towards a one-for-all game library/launcher, that would be fantastic.

**Comment by LuPoNe81** [comment]
I second this. I finally found that I could install GameHub on Manjaro, right after I installed a GOG game through Lutris this morning, and I can't find a way to let GameHub see that game only. It would be a nice addition to GameHub!

**Comment by webodan** [comment]
Yeah, this would be amazing, I hope it's implemented sometime, GameHub is better than any other game controller UI that I could find for Linux, and I have a ton of games on Lutris that would be great to import, so we don't have to manually add tons of games. Please make it happen!

**Comment by denizonm** [comment]
I would also love to see this happen.
